stripe/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsAdapterTest.kt

```diff
@@ -105,18 +106,53 @@ class PaymentOptionsAdapterTest {
     fun `click on Google Pay view should update payment selection`() {
```

Comment:
This test was not really testing the click. The initially selected item was already Google Pay and the click wasn't doing anything because we rely on `adapterPosition` from the ViewHolder, which is only set if the RecyclerView is actually laid out.